src/main/java/com/globant/carrito/cart/Carts.java

```diff
@@ -0,0 +1,134 @@
+package com.globant.carrito.cart;
+
+import java.util.HashSet;
+import java.util.Set;
+import java.util.Date;
+
+import javax.persistence.CascadeType;
+import javax.persistence.Column;
+import javax.persistence.Entity;
+import javax.persistence.EnumType;
+import javax.persistence.Enumerated;
+import javax.persistence.GeneratedValue;
+import javax.persistence.Id;
+import javax.persistence.ManyToOne;
+import javax.persistence.OneToOne;
+
+import com.globant.carrito.product.Items;
+
+
+@Entity
+public class Carts {
+	
+	@Id
+	@GeneratedValue
+	private int cartId;
+	
+	@ManyToOne(cascade = CascadeType.ALL)
+	private int clientId;

```

Comment:
Acá deberia un `Clients client` en lugar de `int clientId`. Acordate que con JPA no necesitás manejar las claves foráneas, te lo hace solo.